extras/basic_example/public/script.js

```diff
@@ -55,66 +63,65 @@ window.onload = function () {
     req.send(JSON.stringify(body));
   };
 
-  createToken("user", "presenter", roomName, function (response) {
+  createToken('user', 'presenter', roomName, function (response) {
     var token = response;
     console.log(token);
     room = Erizo.Room({token: token});
 
-    localStream.addEventListener("access-accepted", function () {
+    localStream.addEventListener('access-accepted', function () {
       var subscribeToStreams = function (streams) {
+        var cb = function (evt){
+            console.log('Bandwidth Alert', evt.msg, evt.bandwidth);
+        };
         for (var index in streams) {
           var stream = streams[index];
           if (localStream.getID() !== stream.getID()) {
             room.subscribe(stream);
-            stream.addEventListener("bandwidth-alert", function (evt){
-                console.log("Bandwidth Alert", evt.msg, evt.bandwidth);
-            });
-            
-
+            stream.addEventListener('bandwidth-alert', cb);
```

Comment:
Creating functions within a loop is discouraged in general: http://airbnb.io/javascript/#functions--in-blocks